token/metadata.go

```diff
@@ -37,6 +40,7 @@ func (m *Metadata) GetToken(raw []byte) (*token2.Token, view.Identity, []byte, e
 	return tok, id, tokenInfoRaw, nil
 }
 
+// SpentTokenID returns the token IDs of the tokens that ware spent by the Token Request this metadata is associated.
```

Comment:
that ware spent by the Token Request this metadata is associated with.

Reply:
done